TransIT.DAL/Models/Entities/WorkType.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using TransIT.DAL.Models.Entities.Abstractions;
+
+namespace TransIT.DAL.Models.Entities
+{
+    public class WorkType : IAuditableEntity, IBaseEntity
+    {
+        public WorkType()
+        {
```

Comment:
So you initializing `ICollection ` in User, but not here. Please, initialize `ICollection `